Sprint-3/revise/implement/get-ordinal-number.js

```diff
@@ -0,0 +1,34 @@
+function getOrdinalNumber(num) {
+    //function will add the correct suffix to the appropriate number with certain conditions
+    let ordinal = "";
+    if (num > 0) {
+        let StringNum = num.toString();
+        //Have to account for 11-19 which end with 'th'
+        if (num % 100 === 11 || num % 100 === 12 || num % 100 === 13) {
+            ordinal = "th";
+        }
+        else if (Number(StringNum[StringNum.length - 1]) === 1 || Number(num) === 1) {
```

Comment:
Why do you need to specifically check for `Number(num) === 1`?

Reply:
I added it as a precaution, not realising that it was already being covered by the condition give. I'll remove that. Thanks for the heads-up. I must have written it out when thinking of the specific conditions to give the if-else statement